src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -323,3 +386,350 @@
+          -r id=942120 -r id=942100
+"""
+
+helps['cdn waf policy managed-rule-set rule-group-override delete'] = """
+type: command
+short-summary: Remove a rule group override from a managed rule set on a CDN WAF policy.
+examples:
+  - name: Remove the rule group override for SQLI from DefaultRuleSet_1.0 on a CDN WAF policy.
+    text: |
+        az cdn waf policy managed-rule-set rule-group-override delete -g group --policy-name policy \\
+          --rule-set-type DefaultRuleSet --rule-set-version 1.0 -n SQLI
+"""
+
+helps['cdn waf policy managed-rule-set rule-group-override list'] = """
+type: command
+short-summary: List rule group overrides of a managed rule on a CDN WAF policy.
+examples:
+  - name: List rule group overrides of a managed rule on a CDN WAF policy.
+    text: >
+        az cdn waf policy managed-rule-set rule-group-override list -g group --policy-name policy \\
+          --rule-set-type DefaultRuleSet --rule-set-version 1.0
+"""
+
+helps['cdn waf policy managed-rule-set rule-group-override show'] = """
+type: command
+short-summary: Show a rule group override of a managed rule on a CDN WAF policy.
+examples:
+  - name: Get the rule group override for rule group SQLI of DefaultRuleSet_1.0 on a CDN WAF policy.
+    text: >
+        az cdn waf policy managed-rule-set rule-group-override show -g group --policy-name policy \\
+          --rule-set-type DefaultRuleSet --rule-set-version 1.0 -n SQLI
+"""
+
+helps['cdn waf policy managed-rule-set rule-group-override list-available'] = """
+type: command
+short-summary: List available CDN WAF managed rule groups of a managed rule set.
+examples:
+  - name: List available rule groups for DefaultRuleSet_1.0.
+    text: |
+      az cdn waf policy managed-rule-set rule-group-override list-available \\
+        --rule-set-type DefaultRuleSet --rule-set-version 1.0
+"""
+
+helps['cdn waf policy managed-rule-set rule-group-override show-available'] = """
+type: command
+short-summary: Show an available CDN WAF managed rule group.
+examples:
+  - name: Show the SQLI rule group of DefaultRuleSet_1.0.
+    text: |
+      az cdn waf policy managed-rule-set rule-group-override show-available \\
+        --rule-set-type DefaultRuleSet --rule-set-version 1.0 -n SQLI
+"""
+
+helps['cdn waf policy custom-rule'] = """
+type: group
+short-summary: Manage custom rules of a CDN WAF policy.
+"""
+
+helps['cdn waf policy custom-rule set'] = """
+type: command
+short-summary: Add a custom rule to a CDN WAF policy.
+parameters:
+  - name: --action
+    type: string
+    short-summary: >
+        The action to take when the rule is matched.
+  - name: --match-condition -m
+    type: string
+    short-summary: Conditions used to determine if the rule is matched for a request.
+    long-summary: >
+        Match conditions are specified as key value pairs in the form "KEY=VALUE [KEY=VALUE ...]".
+        Available keys are 'match-variable', 'operator', 'match-value', 'selector', 'negate', and
```

Comment:
Hi. How can user know the valid values for each key?

Reply:
I've updated the long-summary to include this information.